test/api/v3/integration/user/DELETE-user.test.js

```diff
@@ -30,6 +30,20 @@ describe('DELETE /user', () => {
     });
   });
 
+  it('does not need a password if logged in through Facebook', async () => {
+    await user.auth.facebook.id.del('/user', {
```

Comment:
Thanks for that. Sorry for the late reply to this. I will go ahead and work on the generator. I'll update soon.